app/views/devise/passwords/new.html.haml

```diff
@@ -0,0 +1,11 @@
+%h2 Forgot your password?
+%br
```

Comment:
`<br>` is not intended for margin or spacing in general. Use CSS for this. I think @arbins created some css classes for this cases.